api/proto/config/storage.proto

```diff
@@ -242,3 +242,39 @@ message DisksConfig {
   DisksArrayType array_type = 2;
   repeated DisksConfig children = 3;
 }
+
+// RollbackCmd - snapshot rollback command
```

Comment:
Please add some more comment indicating that if this message is included and the counter in the DeviceOpsCmd is different, it will cause a rollback operation.

Reply:
I added comment